providers/dbt/cloud/src/airflow/providers/dbt/cloud/utils/openlineage.py

```diff
@@ -48,7 +48,7 @@ def _get_logical_date(task_instance):
     return date
 
 
-@require_openlineage_version(provider_min_version="2.0.0")
+@require_openlineage_version(provider_min_version="2.3.0")
```

Comment:
Why 2.3.0 here and 2.2.0 in pyproject.toml ?

Reply:
Fixed. There was a release in the meantime, did not catch all the places.